frontend/src/components/layout.js

```diff
@@ -26,7 +26,7 @@ const Layout = ({ children }) => {
     <div className="bg-Neutral-100 font-raleway">
       <Header siteTitle={data.site.siteMetadata?.title || `Title`} />
       <div>
-        <main className="max-w-container mx-auto">{children}</main>
+        <main className="max-w-full flex flex-col items-center mx-auto bg-Shades-0 py-6">{children}</main>
```

Comment:
Could you change this into below: 
```suggestion
        <main className="flex flex-col items-center bg-Shades-0 py-6">{children}</main>
```

container is already set to full width mx-auto is only needed when having width less to 100% to center the container;

Reply:
Good catch, I will change that!